keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -452,94 +457,192 @@ change are understandable. This may include API specs (though not always
+- A slim base image may be introduced to act as a replacement for [scratch]. On windows graphdriver calls expect some files (mainly registry hives) and these would be included in the slim base image.
+- Privileged containers will not inherit the same [compatibility requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility) as process isolated containers from an OS perspective. Container runtimes like containerd may be able to use fields on `WindowsPodSandboxConfig` to skip OS version checks when pulling/starting these containers in the future. This will continue to be investigated as the feature matures.
+
 #### Container Image Build/Definition
-- This is another ongoing area of investigation and open to feedback. 
- 
+
+- This is another ongoing area of investigation and open to feedback. Currently only a subset of dockerfile operations are supported (ADD, COPY, PATH, ENTRYPOINT, etc).
+
 ### CRI Implementation Details
-We will need to add a privileged field to the runtime spec. We can model this after the Linux pod security context and container security context that is a boolean that is set to “true” for privileged containers. References:
-- [Linux SandboxSecurityConfig](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto#L293)
+
+We will need to add a `hostProcess` field to the runtime spec. We can model this after the Linux pod security context and container security context that is a boolean that is set to `true` for privileged containers. References:
+
+- [LinuxSandboxSecurityConfig](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto#L293)
 - [LinuxSandboxSecurityContext](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto#L28)
 - [LinuxContainerSecurityContext](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto#L612)
 
-In windows, the same privileged field would need to be added in the windows pod security context (open issue) and the container security context will need to be augmented with the privileged flag.
+For Windows we are proposing the following updates to CRI-API:
 
-The new WindowsPodSandboxConfig:
-```
-message WindowsPodSandboxConfig {.
-	WindowsSandboxSecurityContext security_context = 1;
+Add WindowsPodSandboxConfig (and it to PodSandboxConfig)
+
+```protobuf
+message WindowsPodSandboxConfig {
+  WindowsSandboxSecurityContext security_context = 1;
 }
 ```
 
-A new WindowsSandboxSecurityContext:
-```
+Add WindowsSandboxSecurityContext:
+
+```protobuf
 message WindowsSandboxSecurityContext {
-	string run_as_user = 1;
-	bool privileged = 2;
+  string run_as_username = 1;
+  string credential_spec = 2;
+  bool hostProcess = 3;
 }
 ```
 
-The new field for the WindowsContainerSecurityContext:
-```
+Update WindowsContainerSecurityContext by adding privileged field:
```

Comment:
"by adding privileged field" needs to be updated to host_process (also should it be underscored like the others and not camel case?)